app/styles/components/_modal-dialog.scss

```diff
@@ -1,22 +1,20 @@
 // Modals using the ember-modal-dialog addon.
 
 .ember-modal-dialog {
-	padding: 2.4rem 2rem 0.5rem;
+	padding: 2.4rem 2rem 0;
+	box-shadow: $box-shadow;
+	border: 1px solid $mediumlightgray;
 	border-radius: $border-radius;
-	box-shadow: -4px 4px 12px rgba(88, 88, 88, 0.25);
 
-	// forms inside modals
 	.Form {
-		max-width: 100%;
-		margin-bottom: 0;
-
 		@media (min-width: 520px) {
```

Comment:
why don't we use one of our brake point?

Reply:
no reason, will do